broker/image_rhel.yaml

```diff
@@ -58,6 +58,9 @@ artifacts:
     - path: sasl-delegation-0.16.1.rhom-001001-redhat-4.jar
       md5: e8fdae0e7782fb28109681fa3ebca694
       name: sasl-delegation.jar
+    - path: jmx_exporter-0.16.1.rhom-001001-redhat-4.jar
```

Comment:
Added artifact to http://ce-cacher.usersys.redhat.com/ which is required for the artifact to be picked up.  (Had to get the cacher restarted first, it was misbehaving)